### DISCUSSION TITLE
Scanning across ALL of youtube

### DISCUSSION THREAD

**Discussion by RoboPterodactyl** [ThioJoe/YT-Spammer-Purge#1061] [Ideas]
Would it be hard to have the ability to scan EVERY video on youtube?

**Comment by Firecul** [comment]
Yes

**Reply by RoboPterodactyl** [reply]
Do you mean 'Yes. It be hard to have the ability to scan EVERY video on youtube' or do you mean 'Yes. I want this ability'?

**Reply by Firecul** [reply]
I mean unless you have direct access to the YT servers it's impossible.

**Reply by boywithkeyboard** [reply]
😂

**Comment by boywithkeyboard** [comment]
@RoboPterodactyl if you mean a single account, yes and no - you'd probably get ratelimited by the API thus it wouldn't really work. If you mean entire YouTube, then **definitely not possible** (but I think this shouldn't need to be answered anyway 😂).

**Reply by RoboPterodactyl** [reply]
I mean entire YouTube 😂😂😂

**Reply by RoboPterodactyl** [reply]
I did think about the API issue but if ThioJoe makes it use an embedded browser like the [Chromium Embedded Framework](https://github.com/chromiumembedded) that wouldn't be an issue. I did start a poll on this:
https://github.com/ThioJoe/YT-Spammer-Purge/discussions/1062#discussion-5429481
Vote "Yes. Totally abandon the API and switch to using a browser" if you want to fix the API issue.

**Reply by boywithkeyboard** [reply]
@RoboPterodactyl, you can close this thread for good, because what you are trying to do is downright impossible.

**Reply by boywithkeyboard** [reply]
Also, scraping their website is probably against their Terms of Service and it's something they try to defend against according to [this StackOverflow thread](https://stackoverflow.com/questions/30587390/what-is-the-legality-of-scraping-youtube-data).
